### DISCUSSION TITLE
Docker-compose error on Ubuntu Server [May 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2196] [Discord Threads]
> 🙋‍♂️ **jiub**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### connection to db ? can u share more details  
#  
> 🙋‍♂️ **jiub**  
#### I'm starting this docker-compose file as it provided in docs: docker-compose/mysql/docker-compose.yml  
#  
> 👨‍🚀 **wingkwong**  
#### can u share the full error message  
#  
> 🙋‍♂️ **jiub**  
#### https://pastebin.com/YHtFjkas  
#  
> 🙋‍♂️ **jiub**  
#### And after startup root_db spamming this line: root_db_1  | 2022-05-20T09:59:39.198738Z 5 [Note] Access denied for user 'root'@'localhost' (using password: NO)  
#  
> 👨‍🚀 **wingkwong**  
#### are u sure the password s correct  
#  
> 👨‍🚀 **wingkwong**  
#### is the db on yourh host machine?  
#  
> 🙋‍♂️ **jiub**  
#### No, db starts inside docker container  
#  
> 🙋‍♂️ **jiub**  
#### I use this docker-compose https://github.com/nocodb/nocodb/blob/develop/docker-compose/mysql/docker-compose.yml  
#  
> 🙋‍♂️ **jiub**  
#### system info: https://pastebin.com/5MgLYDiP  
#  
> 🙋‍♂️ **jiub**  
#### docker run -d --name nocodb-mysql \   
#### -v "$(pwd)"/nocodb:/usr/app/data/ \   
#### -p 8080:8080 \   
#### -e NC_DB="mysql2://host.docker.internal:3306?u=root&p=password&d=d1" \   
#### -e NC_AUTH_JWT_SECRET="569a1821-0a93-45e8-87ab-eb857f20a010" \   
#### nocodb/nocodb:latest   
####    
####    
#### this command also not working on Ubuntu 20.04  
#  
> 🙋‍♂️ **jiub**  
#### I get this error: https://pastebin.com/Th9xrW2i  
#  
> 👨‍🚀 **wingkwong**  
#### for this one, u need a db up n running on your host machine  
#  
> 👨‍🚀 **wingkwong**  
#### try to remove all the images & volumes n run it again  
#  
> 🙋‍♂️ **jiub**  
#### Still not working. My requests to nocodb doesn't shows in docker-compose logs.  
#  
> 🙋‍♂️ **jiub**  
#### Ok now I'm trying to deploy via nginx. Can you help me with 'path/to/webroot' inside docker-compose file? What I'm must specify instead of path/to/webroot?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/977135481710067712)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
